<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSetting</class>
 <widget class="QDialog" name="ControlSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>312</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AR PL UKai CN</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>仪表控制</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../src/image.qrc">
    <normaloff>:/images/leiwo.png</normaloff>:/images/leiwo.png</iconset>
  </property>
  <widget class="QComboBox" name="cbx_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>停止</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>动态展开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>静态展开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>收回</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>39</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/UnloaderSetting.png);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_chrs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/parameters/CutHeightResetOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>脱开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>割台提升</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>割台下降</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="m_reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/engine/ZeroOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;AR PL UKai CN&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>清零</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="auto_load_regu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/con_recong/LoadRegulationAutoOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>关闭负荷自动调节</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>开启负荷自动调节</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="auto_height">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/CutHeightAutoOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>关闭割台高度自动调节</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>开启割台高度自动调节</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="l_rw_cl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/WeightClearOn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>清零</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbx_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无动作</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>静音</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>播放</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="m_reset_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/warning/on.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_btn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 88, 88);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_btn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 111, 249);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_103">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>59</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/con_recong/ProfilingHeightSet.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_pfh">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_102">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/Weight1000.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_weight1000">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>119</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/RollerSpeedLowSetting.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>109</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/cut_height_min.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/cut_height_max.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_rsls">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_cuth_max">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_cuth_min">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>70</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/ConcaveClearanceSetting.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_concaveclset">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>39</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/CutMachineMode.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_105">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/engine/RiceKind.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="grain_kind">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>水稻</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小麦</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>玉米</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大豆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>其他</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cutterMode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>切碎</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>排草</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>41</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/ConcaveClearanceMax.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/ConcaveClearanceMin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_heartbeat">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>仪表在线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>仪表掉线</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/heartbeat.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/GrainMoisture.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_ConcaveclSetMin">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_ConcaveclSetMax">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/GrainDensity.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Kg/m3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/GrassGrainRatio.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_GrassrGrainratio">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_PlantHeight">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/PlantHeight.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/AxialRollerSpeedMin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_AxialRollerSpeedMin">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/AxialRollerSpeedMax.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_AxialRollerSpeedMax">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_StalkMoisture">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_CroppingIntensity">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SoftKeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>softkeylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../src/image.qrc"/>
 </resources>
 <connections/>
</ui>
